### ISSUE TITLE
qvalues are always smaller than the pvalue

### ISSUE THREAD

**Issue by jazberna1** [koszullab/chromosight#40]
Hello Cyril,

I have noticed that in my tsv output for chromosight v.1.4.0 the qvalues are always smaller than the pvalues. I would expect the opposite.  Should I still filter based on the qvalues?

These are the first lines of my output:
chrom1  start1  end1    chrom2  start2  end2    bin1    bin2    kernel_id       iteration       score   pvalue  qvalue
1       9800000 9900000 1       9900000 10000000        98      99      0       0       0.3970866534    0.0000000617    
1       12300000        12400000        1       12500000        12600000        123     125     0       0       0.4051226690    0.0000002507    
1       20300000        20400000        1       20400000        20500000        203     204     0       0       0.4270293101    0.0000000041    
1       33800000        33900000        1       34000000        34100000        338     340     0       0       0.3583274096    0.0000004544    
1       39400000        39500000        1       39500000        39600000        394     395     0       0       0.4120016497    0.0000000167  

This is how I run chromosight:

```
/mnt/lustre/scratch/SOFTWARE/miniconda3/bin/chromosight \
detect \
-z 100 -u 100 \
--threads=1 \
--inter \
/mnt/lustre/scratch/results.test/matrix/test.25000.cool \
loops_test_25000
```

Thanks so much
Jorge

**Comment by cmdoret** [comment]
Hi Jorge, 

If I take the first line in your example, 0.39 is the "score" column. It is the pearson correlation between this window and the kernel.  the last column with value 0.0000000617 is the p-value. The q-value column is empty.

I am not sure why the q-value column is empty, and I'll have a look at the code, but q-values are not necessary anyway. I would advise filtering based on scores.

**Comment by jazberna1** [comment]
Hello Cyril,

Thanks so much. I'll review the scores as you say.

Regards
Jorge

**Comment by cmdoret** [comment]
OK I fixed the bug in c5fee1fb640cca8927f4b338aa98c55bb0a5282d. It happened only when `-z` and `-u` were set to 100 and escaped the tests. If you want your q-values, you can update to 1.4.1 if you want, or just set `-z` and `-u` to something more conservative like 80 and it should work as well.

P.S: 1.4.1 should be available in a few minutes

Cheers,
Cyril

**Comment by jazberna1** [comment]
Hello Cyril,

Thanks a for the fix!

Jorge
